### DISCUSSION TITLE
version node docusaurus 2.0.0-beta.21

### DISCUSSION THREAD

**Discussion by rafaelrdealmeida** [facebook/docusaurus#7546] [Q&A] [answered]
Hi,

we are upgrading beta.21 and render.com tells us:

```
Jun 1 04:13:09 PM  ==> Running build command 'cd website; yarn install; yarn build'...
Jun 1 04:13:09 PM  yarn install v1.22.18
Jun 1 04:13:09 PM  warning ../package.json: No license field
Jun 1 04:13:09 PM  [1/5] Validating package.json...
Jun 1 04:13:09 PM  error website@0.0.0: The engine "node" is incompatible with this module. Expected version ">=16". Got "14.19.1"
Jun 1 04:13:09 PM  error Found incompatible module.
Jun 1 04:13:09 PM  info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Jun 1 04:13:09 PM  yarn run v1.22.18
Jun 1 04:13:09 PM  warning ../package.json: No license field
Jun 1 04:13:09 PM  error website@0.0.0: The engine "node" is incompatible with this module. Expected version ">=16". Got "14.19.1"
Jun 1 04:13:09 PM  error Commands cannot run with an incompatible environment.
```

it's a problem in the nodejs version. We use render.com. It is possible to solve this by following the instructions in the link below:

https://render.com/docs/node-version

the render use by default 14.17.0

I saw that on azure this also happens (see https://github.com/Azure/static-web-apps/issues/694):

https://blog.johnnyreilly.com/2022/05/28/azure-static-web-apps-node-16-oryx#the-engine-node-is-incompatible

this is a deploy specific problem or docusaurus could solve (I believe other users should go through this)

best

**Comment by slorber** [comment, marked as answer]
As the message explains (and also our changelog breaking changes + release tweet): upgrade Node to >= v16.14
